distribution/org.wso2.developerstudio.eclipse.distribution.project/src/org/wso2/developerstudio/eclipse/distribution/project/ui/wizard/ProjectImportWizard.java

```diff
@@ -72,6 +76,8 @@ private void searchAndRemoveGraphicalSynapseCongif(IProject project) {
 		try {
 			if (project.hasNature("org.wso2.developerstudio.eclipse.esb.project.nature")) {
 				deleteGraphicalSynapseConfigDir(project);
+			} else if (project.hasNature("org.wso2.developerstudio.eclipse.general.project.nature")){
```

Comment:
We can use defined constants for project natures at org.wso2.developerstudio.eclipse.platform.core.utils.Constants class